dev-db/sqldeveloper/sqldeveloper-20.4.1.407.0006.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop java-pkg-2
+
+DESCRIPTION="Oracle SQL Developer is a graphical tool for database development"
+HOMEPAGE="https://www.oracle.com/technetwork/developer-tools/sql-developer/overview/index.html"
+SRC_URI="${P}-no-jre.zip"
+
+RESTRICT="fetch"
+
+LICENSE="OTN"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+IUSE="mssql mysql postgres sybase"
+
+RDEPEND="mssql? ( dev-java/jtds:1.3 )
+	mysql? ( dev-java/jdbc-mysql:0 )
+	postgres? ( dev-java/jdbc-postgresql:0 )
+	sybase? ( dev-java/jtds:1.3 )
+	>=dev-java/openjdk-8:*[javafx]
+	>=virtual/jre-1.8:*"
```

Comment:
We usually sort unconditional deps before conditional ones, 
https://devmanual.gentoo.org/general-concepts/dependencies/
(just FYI)

Reply:
Next release I change order, thank you.